### PR TITLE
v5.10 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Fixed an issue where a channel did not get removed from the unreads section if the user navigated out of it via a permalink.
+ - Fixed an issue where a link from Access Control Groups to Group Filter on AD/LDAP did not work for subpath Site URL.
+ - Fixed an issue where expired channels appeared in "My Channels" section of channel switcher if using the **Automatically Close Direct Messages** setting.
+ - Fixed an issue where the text box reverted to default size after a user returned from the Integrations page.
+ - Fixed an issue where the profile popover wasn't allowed to close itself when opened through an at-mention.
+ - Fixed an issue where filtering by first name with Korean characters no longer worked for at-mentions.
+ 
+### Compatibility
+
+ - Deprecated configurable ``timezones.json`` in favour of the existing hard-coded list built into the server. **XXXX NEEDS MORE DETAILS**
```
Comment (amyblais) [inline source/administration/changelog.md]: Is this a deprecated feature or just part of the Configuration in Database?

https://mattermost.atlassian.net/browse/MM-14260

Comment (cpanato) [reply source/administration/changelog.md]: @lieut-data please comment here

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### API Changes
+ - Added ``GetUsers`` API method to add the ability to list users.
+ - Added the ``SearchPostsInTeam`` method to the plugin API to add the ability to search posts in a team.
+ - Added ``GetTeamMembersForUser`` and ``GetChannelMembersForUser`` to the plugin API to add the ability to get team and channel members for a specific user.
+ - Added ``GetBundleInfo() string`` method to the plugin API to add the ability to store assets elsewhere.
+ 
+### Websocket Event Changes
+
+### Database Changes
```
Comment (cpanato) [inline source/administration/changelog.md]: added bots roles/permissions, seel: https://github.com/mattermost/mattermost-server/pull/10507/files

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+### Highlights
+
+#### Infinite One Directional Scrolling
+ - Removed the “Load more messages” button at the top of the screen to make it easier to read messages.
+ 
+#### Interactive Ephemeral Messages
+ - Added ability to modify message attachments, allowing for richer integrations.
+
+#### Configuration in Database
+ - Added support for managing configuration options directly in the database. This option honors the current ``config.json`` to maintain backwards compatibility while providing a way to easily manage configuration settings from the system console in more complex environments.
```
Comment (lieut-data) [inline source/administration/changelog.md]: Propose
```suggestion
 - Added experimental support for storing ``config.json`` in the database, improving the system console experience on read-only filesystems. Storing the configuration in the database is optional, as the existing ``config.json`` remains fully supported.
```

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Created CLI command ``webhook show`` to allow seeing detailed information of a webhook.
+ - Created CLI command ``team rename`` to allow renaming teams.
+ - Created CLI command ``channel search`` to allow searching for channels.
+ 
+#### Administration
+ - Improved default session timeout behavour, including changing the default ``SessionLengthWebInDays`` from 30 to 180 days.
+ - Added full text search to the system console panel to easily find options in the configuration.
+ - (Advanced Permissions) Split managing emoji permissions into "create", "delete own" and "delete others".
+ - (Advanced Permissions) Added ``List_Public_Teams``, ``Join_Public_Teams``, ``List_Private_Teams`` and ``Join_Private_Teams`` permissions.
+ - Updated Mattermost to default to console logging in a human readable format.
```
Comment (lieut-data) [inline source/administration/changelog.md]: This only affects developer mode, and not production: should we omit it from the release notes or qualify it as such?

File: source/administration/changelog.md (hunk 1 above)
Comment (lieut-data) [reply source/administration/changelog.md]: I think the description (minus the `XXXXX` part ;P)) is sufficient: it's independent of the configuration in the database changes.

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Deprecated configurable ``timezones.json`` in favour of the existing hard-coded list built into the server. **XXXX NEEDS MORE DETAILS**
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+ 
+#### Changes to Team Edition and Enterprise Edition:
+
+- Under ``"ExperimentalSettings":`` in ``config.json``:
+   - Added ``"RestrictSystemAdmin": false``, to
```
Comment (lieut-data) [inline source/administration/changelog.md]:
```suggestion
   - Added ``"RestrictSystemAdmin": false``, to optionally constrain even system admins from changing critical settings.
```

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+Release Date 2019-04-16
+
+### Highlights
+
+#### Infinite One Directional Scrolling
+ - Removed the “Load more messages” button at the top of the screen to make it easier to read messages.
+ 
+#### Interactive Ephemeral Messages
+ - Added ability to modify message attachments, allowing for richer integrations.
```
Comment (jasonblais) [inline source/administration/changelog.md]:
```suggestion
 - Added support for [message actions and buttons](https://docs.mattermost.com/developer/interactive-messages.html) in ephemeral messages.
```

File: source/administration/changelog.md (hunk 2 above)
Comment (jasonblais) [reply source/administration/changelog.md]: Propose

```
Granted the following permissions for the System Admin, in preparation for an upcoming release of bot accounts:
 - PERMISSION_CREATE_BOT
 - PERMISSION_READ_BOTS
 - PERMISSION_READ_OTHERS_BOTS
 - PERMISSION_MANAGE_BOTS
 - PERMISSION_MANAGE_OTHERS_BOTS
```

File: source/administration/changelog.md (hunk 4 above)
Comment (amyblais) [reply source/administration/changelog.md]: Removed,

File: source/administration/changelog.md [hunk 7]
```diff
@@ -54,55 +48,75 @@ Release Date 2019-04-16
...
+   - PERMISSION_READ_BOTS
+   - PERMISSION_READ_OTHERS_BOTS
+   - PERMISSION_MANAGE_BOTS
+   - PERMISSION_MANAGE_OTHERS_BOTS
 
 ### Known Issues
+ - On a server using a subpath, the URL opens a blank page if the system admin changes the Site URL in the System Console UI. The system admin should restart the server to fix it.
+ - Login does not work when Custom Terms of Service is enabled and MFA is enforced.
+ - Google login fails on the Classic mobile apps.
+ - User can receive a video call from another browser tab while already on a call.
```
Comment (lieut-data) [inline source/administration/changelog.md]: To what is this referring?

Comment (lieut-data) [reply source/administration/changelog.md]: Specifically:
> - User can receive a video call from another browser tab while already on a call.

Comment (amyblais) [reply source/administration/changelog.md]: I wasn't able to find a Jira ticket so I removed it from the list for now,

Comment (jasonblais) [reply source/administration/changelog.md]: PS: This was an issue with the legacy WebRTC calls

**Pull request merged**